server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultClusterContext.java

```diff
@@ -992,17 +759,48 @@ private void initializeCluster(Config config) {
             this.collectionsFactory = new ClusterCollectionsFactory(clusterInstance);
             this.messageBufferFactory = new ClusterMemoryMessageBufferFactory(clusterInstance);
             localNodeId = getMemberId(cluster.getLocalMember());
-            clusterMessaging = new ClusterMessaging(this, clusterInstance, schedulerProvider);
 
             IMap<MemberId, String> instanceKeyMap = collectionsFactory.getMap(INSTANCE_KEY_MAP);
             instanceKeyMap.put(localNodeId, localInstanceKey);
-            instanceKeyMap.addEntryListener(instanceKeyEntryListener, true);
 
             IMap<String, Collection<String>> balancerMap = collectionsFactory.getMap(BALANCER_MAP_NAME);
-            balancerMap.addEntryListener(balancerMapEntryListener, true);
+            addBalancerMapEntryListeners(balancerMap);
         }
     }
 
+    private void addBalancerMapEntryListeners(IMap<String, Collection<String>> balancerMap) {
```

Comment:
This is what I meant when I said that the logging statements are still there. I removed the firing of the BalancerMapListeners from this Hazelcast map listener, but left the logging of changes.